system/jlib/jptree.cpp

```diff
@@ -4689,17 +4750,20 @@ template <typename T>
 class CommonReaderBase : public CInterface
 {
     Linked<ISimpleReadStream> lstream;
-    ISimpleReadStream *stream;
-    bool bufOwned, nullTerm;
-    byte *buf, *bufPtr;
-    size32_t bufSize, bufRemaining;
+    ISimpleReadStream *stream{nullptr};
+    bool bufOwned{false};
+    bool nullTerm{false};
+    byte *buf{nullptr};
+    byte *bufPtr{nullptr};
+    size32_t bufSize{0};
+    size32_t bufRemaining{0};
```

Comment:
it's fine, but I think better not to make auxillar unrelated changes to classes that are not related to changes in a PR, even if they are good/minor like this. It conflates the review.